source/views/transportation/other-modes/other-modes-detail.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React from 'react'
+import {StyleSheet} from 'react-native'
+import {Markdown} from '../../components/markdown'
+import {ListFooter} from '../../components/list'
+import glamorous from 'glamorous-native'
+import {tracker} from '../../../analytics'
+import {Button} from '../../components/button'
+import openUrl from '../../components/open-url'
+import type {OtherModeType} from '../types'
+
+const AAO_URL = 'https://github.com/StoDevX/AAO-React-Native/issues/new'
+
+const Title = glamorous.text({
+  fontSize: 36,
+  textAlign: 'center',
+  marginHorizontal: 18,
+  marginVertical: 10,
+})
+
+const Container = glamorous.scrollView({
+  paddingHorizontal: 18,
+  paddingVertical: 6,
+})
+
+const styles = StyleSheet.create({
+  paragraph: {
+    fontSize: 16,
+  },
+})
+
+export class OtherModesDetailView extends React.PureComponent {
+  static navigationOptions = ({navigation}) => {
+    return {
+      title: navigation.state.params.mode.name,
+    }
+  }
+
+  props: {navigation: {state: {params: {mode: OtherModeType}}}}
+
+  onPress = () => {
+    const {name, url} = this.props.navigation.state.params.mode
+    tracker.trackScreenView(
```

Comment:
What does this do if there's no URL to open? Should this be moved into the If-block?

Reply:
Actually, why didn't you use trackedOpenUrl?